src/ray/core_worker/test/direct_task_transport_test.cc

```diff
@@ -112,6 +112,10 @@ class MockRayletClient : public WorkerLeaseInterface {
     return Status::OK();
   }
 
+  ray::Status ReleaseUnusedWorkers(const std::vector<WorkerID> &used_workers) override {
```

Comment:
`used_workers` is a little wired, how about `exclude_workers`?

Reply:
fixed